Artifacts/semver.artifactbundle/info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "schemaVersion": "1.0",
+    "artifacts": {
+        "semver": {
+            "version": "3.4.0",
+            "type": "executable",
+            "variants": [
+                {
+                    "path": "semver-3.4.0-any/bin/semver.sh",
+                    "supportedTriples": ["x86_64-apple-macosx", "arm64-apple-macosx", "x86_64-unknown-linux-gnu"]
```

Comment:
I'm not totally sure about the Linux variant, but it's good enough to work on the default GH Ubuntu runner